CommandLine/emake/Server.cpp

```diff
@@ -81,11 +83,25 @@ class CompilerServiceImpl final : public Compiler::Service {
         std::string name = about.get("name");

```

Comment:
Why aren't you just using the existing reflection-based YAML parse infrastructure for this?

Reply:
Ok, that's a good suggestion Josh, but if I do that I would have to include the egm header which makes this depend on `CLI_ENABLE_EGM` which would be fine, because the server obviously depends on libEGM, but it would make `OptionsParser::printInfo` also depend on libEGM. I don't want to extra the yaml-proto reflection from the egm reader right now either because that will introduce conflicts on the unfinished egm prs. So for now, I just want to do it this way and we can come back to it.

Reply:
Or you can move it to Util where everything's already including from.

Reply:
As we discussed over Discord, I am the one who created the Util header you're referring to. That was done on the egm branch that isn't merged yet. So, yeah, let's expend a little technical debt here and I'll come back to do that later.